x-pack/plugins/alerting/server/alerts_client/alerts_client.test.ts

```diff
@@ -720,40 +467,157 @@ describe('Alerts Client', () => {
                       tags: ['rule-', '-tags'],
                       uuid: '1',
                     },
-                    start: '2023-03-28T12:27:28.159Z',
+                    start: date,
                     status: 'active',
                     time_range: {
-                      gte: '2023-03-28T12:27:28.159Z',
+                      gte: date,
                     },
-                    uuid: 'abc',
+                    uuid: uuid2,
                     workflow_status: 'open',
                   },
                   space_ids: ['default'],
-                  version: '8.8.0',
+                  version: '8.9.0',
                 },
                 tags: ['rule-', '-tags'],
               },
+            ],
+          });
+        });
+
+        test('should update ongoing alerts in existing index', async () => {
+          clusterClient.search.mockResolvedValue({
+            took: 10,
+            timed_out: false,
+            _shards: { failed: 0, successful: 1, total: 1, skipped: 0 },
+            hits: {
+              total: {
+                relation: 'eq',
+                value: 1,
+              },
+              hits: [
+                {
+                  _id: 'abc',
+                  _index: '.internal.alerts-test.alerts-default-000001',
+                  _seq_no: 41,
```

Comment:
Here we see one of the changes to existing behavior.  When querying for previous alerts, we also get the seqNo/primaryTerm info via some new option.  We need these for optimistic concurrency control (OCC) updates to alert docs, with data streams.  Previously, we could just bulk `index` updates to alerts, now we need to provide OCC info as well ...